src/config/routes.ts

```diff
@@ -17,7 +17,7 @@ export const AppRoutes = {
     modules: '/settings/modules',
     index: '/settings',
     appearance: '/settings/appearance',
-    safeAppsPermissions: '/settings/safe-apps-permissions',
```

Comment:
Did you change this by hand? Because it's supposed to generate a nested object here.

Reply:
Could you please mod the script to camelCase dash-separated paths?